lib/Mojo/WebSocketProxy/RequestLogger.pm

```diff
@@ -0,0 +1,91 @@
+use Object::Pad;
+
+class Mojo::WebSocketProxy::RequestLogger;
+
+use strict;
+use warnings;
+use Log::Any qw($log);
+use UUID::Tiny;
+
+field $context;
+
+our $default_handler = sub {
+    my ($level, $message, $context, @params) = @_;
+
+    if(scalar @params) {
+        return $log->$level($message, @params);
+    }
+
+    return $log->$level($message, $context);
+};
+
+sub set_handler {
```

Comment:
Can we initialise it during object construction. Do we really need to change the handler?

Reply:
The `RequestLogger` resides in `WebsocketProxy` which is a public module. We don't want to tie it up to our special `DERIV` logging handling logic.
The object will be created on each message and we don't really need to change the handler. setting it once on start-up of the worker is enough.